docs/.mlc_config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ignorePatterns": [
+        {
+            "pattern": "^http(s)?://localhost(:\\d+)?",
+            "pattern": "^https://opensource.org/licenses/MIT$",
```

Comment:
I don't know why, but opensource.org always throws an 503 error when running in the pipeline. Running markdown link checker locally works as expected without this exception...